tests/unit/components/sl-calendar-test.js

```diff
@@ -270,6 +254,13 @@ test( 'Months for year view are generated validly', function( assert ) {
     );
 });
 
+skip( 'monthsInYearView - active month set correctly', function() {
+    /* Expand preceeding test to also check that the active month is set correctly.
+    https://github.com/softlayer/sl-ember-components/
```

Comment:
Please remove, as IBM's code auditing tools will flag a reference to a url in a comment.  Plus the test name should be enough direction and if it's not then it needs to be made more precise.